<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initialWidget</class>
 <widget class="QWidget" name="initialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>哈夫曼树初始化窗口</string>
  </property>
  <widget class="QSpinBox" name="spinBoxCharSize">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>130</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字符集大小</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInitial">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>初始化哈夫曼树</string>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChooseFile">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>选择待统计字符的文本文件</string>
   </property>
   <property name="whatsThis">
    <string>选择</string>
   </property>
   <property name="text">
    <string>选择文件并统计</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>待统计文件：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatisticedFile">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>存储统计文件：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStoreStatisticFile">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>371</width>
     <height>401</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>在此处显示n个字符及n个权值，每个字符和权值以空格隔开，每对字符与权值以换行符隔开</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>351</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>字符</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>频数</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
